### PR TITLE
addline

### PR DESCRIPTION
**Pull request by MoNaser1**

### CODE REVIEW

File: 3.html [hunk 1]
```diff
@@ -7,6 +7,7 @@
     <style>
         header {
             background-color: lightblue;
+
```
Comment (yousefhou) [inline 3.html:10]: remove this line

**Review by yousefhou — CHANGES_REQUESTED** [review]

File: 3.html (hunk 1 above)
Comment (MoNaser1) [reply 3.html:10]: msh fade

Comment (yousefhou) [reply 3.html:10]: kol tbn

Comment (yousefhou) [reply 3.html:10]: jk

Comment (MoNaser1) [reply 3.html:10]: please approve ya kalb

**Review by yousefhou — APPROVED** [review]

File: 3.html (hunk 1 above)
Comment (yousefhou) [inline 3.html:10]:
```suggestion
           
           
```

**Review by yousefhou — APPROVED** [review]

**Pull request merged**